libs/data-types/src/lib/enums/getEnumMetadata.ts

```diff
@@ -39,6 +40,25 @@ const enumNameToMetadata: Record<string, EnumMetadata> = {
             },
         ],
     },
+    [CoscradEnum.BibliographicSubjectCreatorType]: {
+        type: CoscradDataType.Enum,
+        enumName: CoscradEnum.BibliographicSubjectCreatorType,
+        enumLabel: 'Creator Type',
+        labelsAndValues: [
+            {
+                label: 'artist',
+                value: BiblographicSubjectCreatorType.artist,
```

Comment:
(sp) `BiblographicSubjectCreatorType` should be `BibliographicSubjectCreatorType`

Reply:
Nice catch! Fixed.